crates/re_viewer/src/ui/welcome_screen/example_page.rs

```diff
@@ -0,0 +1,391 @@
+use egui::{NumExt, Ui};
+use re_log_types::LogMsg;
+use re_smart_channel::ReceiveSet;
+use re_ui::ReUi;
+use re_viewer_context::SystemCommandSender;
+use std::collections::HashMap;
+
+// TODO(emilk/egui#3291): replace this by loading image from the web
```

Comment:
We're very close to merging https://github.com/emilk/egui/pull/3297

Reply:
done